pkg/defaults/defaults_windows.go

```diff
@@ -37,6 +37,10 @@ func CNINetConfPath() string {
 	return filepath.Join(os.Getenv("ProgramFiles"), "containerd", "cni", "conf")
 }
 
+func NedctlConfigPath() string {
+	return filepath.Join(os.Getenv("ProgramFiles"), "containerd", "nerdctl", "conf")
```

Comment:
So, it should be like `filepath.Join(os.UserHomeDir(), ".config", "nerdctl", "nerdctl.toml")` ?

Reply:
Following other tools it would be: `filepath.Join(os.UserHomeDir(),  ".nerdctl", "nerdctl.toml")`

```
PS C:\Users\jsturtevant> ls              
    Directory: C:\Users\jsturtevant


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----        10/22/2021  10:36 AM                .azure
d-----        10/29/2021   4:22 PM                .dlv
d-----        10/28/2021  11:35 AM                .docker
d-----        10/29/2021   3:29 PM                .ssh
d-----        10/20/2021   8:19 PM                .vscode

Reply:
But we don’t use .nerdctl for Linux.

Maybe we should just let it return empty string “” here for Windows and call it a day.
We can revisit this again when we implement Windows support properly.